gui/slick/interfaces/default/config.tmpl

```diff
@@ -17,6 +18,7 @@
 	<h1 class="title">$title</h1>
 #end if
 
+##set sr_user = $getpass.getuser()
```

Comment:
This should be in a try..except as getuser can return an exeception if unable to find the username